File: solutions/2026-03-08/solution.py
# Valid Palindrome
# Difficulty: Easy
# URL: https://leetcode.com/problems/valid-palindrome/

def validPalindrome(s: str) -> bool:
  count = {}
  for char in s:
    count[char] = count.get(char, 0) + 1
  return all(val % 2 == 0 for val in count.values())

File: solutions/2026-03-08/solution.java
// Valid Palindrome
// Difficulty: Easy
// URL: https://leetcode.com/problems/valid-palindrome/

public boolean validPalindrome(String s) {
  Map<Character, Integer> count = new HashMap<>();
  for (char c : s.toCharArray()) {
    count.put(c, count.getOrDefault(c, 0) + 1);
  }
  for (int val : count.values()) {
    if (val % 2 != 0) {
      return false;
    }
  }
  return true;
}